go.mod

```diff
@@ -3,15 +3,17 @@ module github.com/crossplane-contrib/terrajet
 go 1.16
 
 require (
-	github.com/crossplane/crossplane-runtime v0.14.0
+	github.com/crossplane/crossplane-runtime v0.15.0
+	github.com/google/go-cmp v0.5.6
+	github.com/hashicorp/hcl/v2 v2.8.2 // indirect
```

Comment:
Is this dependency necessary? I removed it and `make reviewable` still worked.

Reply:
Removed, I am not sure how it ended there but `go mod tidy` does not remove it either.